packages/interfaces/src/connection_manager.ts

```diff
@@ -28,14 +28,18 @@ export enum EPeersByDiscoveryEvents {
   PEER_DISCOVERY_BOOTSTRAP = "peer:discovery:bootstrap",
   PEER_DISCOVERY_PEER_EXCHANGE = "peer:discovery:peer-exchange",
   PEER_CONNECT_BOOTSTRAP = "peer:connected:bootstrap",
-  PEER_CONNECT_PEER_EXCHANGE = "peer:connected:peer-exchange"
+  PEER_CONNECT_PEER_EXCHANGE = "peer:connected:peer-exchange",
+  NODE_ONLINE = "waku:online",
+  NODE_OFFLINE = "waku:offline"
 }
 
 export interface IPeersByDiscoveryEvents {
   [EPeersByDiscoveryEvents.PEER_DISCOVERY_BOOTSTRAP]: CustomEvent<PeerId>;
   [EPeersByDiscoveryEvents.PEER_DISCOVERY_PEER_EXCHANGE]: CustomEvent<PeerId>;
   [EPeersByDiscoveryEvents.PEER_CONNECT_BOOTSTRAP]: CustomEvent<PeerId>;
   [EPeersByDiscoveryEvents.PEER_CONNECT_PEER_EXCHANGE]: CustomEvent<PeerId>;
+  [EPeersByDiscoveryEvents.NODE_ONLINE]: CustomEvent<void>;
```

Comment:
What do you think about it being single event - `NetworkStatus` or something?

Reply:
updated to use a single event of type `CustomEvent<boolean>` with `true` indicating online, false `offline`